pubspec.yaml

```diff
@@ -4,8 +4,22 @@ publish_to: 'none'
 version: 0.1.0
 
 environment:
-  sdk: '>=3.4.0 <4.0.0'
-  flutter: ">=3.24.0"
+  sdk: '>=3.6.0 <4.0.0'
+  flutter: ">=3.27.0"
+
+workspace:
+  # Feature Packages
+  - packages/breez_preferences
+  - packages/sdk_connectivity_cubit
+  - packages/device_client
+  - packages/keychain
+  - packages/lightning_links
+  - packages/notifications_client/firebase_notifications_client
+  # Utility Packages
+  - packages/breez_logger
+  - packages/breez_sdk_liquid
+  - packages/credentials_manager
+  - packages/service_injector
```

Comment:
Should `deep_link_client` be listed here also?

Reply:
No, `deep_link_client` is not listed because it is not currently utilized by the app.